devito/symbolics/extended_sympy.py

```diff
@@ -701,6 +696,31 @@ def rfunc(func, item, *args):
 }
 
 rfunc_mapper = {
-    min: MIN,
-    max: MAX,
+    min: Min,
+    max: Max,
 }
+
+
+def integer_args(*args):
```

Comment:
Isn't this name a bit misleading? Can we use sth better?

Reply:
What do you mean , misleading? It check if the args are integer

Reply:
maybe `has_integer_args` -- but nitpicking here

Reply:
yEAH, sth like has_....check....test...integer_args..

Reply:
done